backup/main/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/featureform/backup"
+	help "github.com/featureform/helpers"
+	clientv3 "go.etcd.io/etcd/client/v3"
+)
+
+func main() {
+	etcdHost := help.GetEnv("ETCD_HOSTNAME", "localhost")
```

Comment:
Not worth changing, but I like allowing people to pass args into it if it's gonna work like a CLI tool instead of setting a ton of env variables.